vnaddress/functions/spell_correction_layer/match_word.py

```diff
@@ -9,7 +9,7 @@ def match_word(error_dict, word_to_check, standard_single_names):
         name_to_check = word_to_check
         # standard_single_names_lower = [name.lower() for name in standard_single_names]
         if name_to_check not in standard_single_names:
-            extraction = (process.extractOne(name_to_check, error_dict[key], scorer=fuzz.ratio))
+            extraction = process.extractOne(name_to_check, error_dict[key], scorer=fuzz.ratio)
             if extraction[1] > weight:
```

Comment:
when performance is important it makes sence to use:
```python
extraction = process.extractOne(name_to_check, error_dict[key], scorer=fuzz.ratio, score_cutoff=weight)
if extraction and extraction[1] > weight:
```
This way it can exit early on matches below this score_cutoff